src/send/saga.ts

```diff
@@ -303,14 +287,72 @@ export function* sendPaymentSaga({
   }
 }
 
+export function* sendTransactionsSaga(
+  serializablePreparedTransactions: SerializableTransactionRequest[],
+  networkId: NetworkId,
+  createBaseStandbyTransactions: ((
+    hash: string,
+    feeCurrencyId?: string
+  ) => BaseStandbyTransaction)[]
+) {
+  const network = getNetworkFromNetworkId(networkId)
+  if (!network) {
+    throw new Error('Unknown token network')
+  }
+
+  const wallet = yield* call(getViemWallet, networkConfig.viemChain[network])
+  if (!wallet.account) {
+    // this should never happen
+    throw new Error('no account found in the wallet')
+  }
+
+  // Unlock account before executing tx
+  yield* call(getConnectedUnlockedAccount)
+
+  // @ts-ignore typed-redux-saga erases the parameterized types causing error, we can address this separately
+  let nonce: number = yield* call(getTransactionCount, wallet, {
+    address: wallet.account.address,
+    blockTag: 'pending',
+  })
+
+  const preparedTransactions = getPreparedTransactions(serializablePreparedTransactions)
+  const txHashes: Hash[] = []
+  for (let i = 0; i < preparedTransactions.length; i++) {
+    const preparedTransaction = preparedTransactions[i]
+    const createBaseStandbyTransaction = createBaseStandbyTransactions[i]
```

Comment:
We should check `createBaseStandbyTransaction` is not undefined and throw.
To prevent incorrect use of the saga.

This reminds maybe we should consider enabling [noUncheckedIndexedAccess](https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess).
I'll try to bring this up at the next eng meeting.